docs/guides/7-sharing-rulesets.md

```diff
@@ -116,11 +116,14 @@ extends:
   - example-spectral-ruleset
 ```
 
-Pegging a ruleset on given version is possible:
+Pegging a ruleset on given version is possible through a `package.json`:
 
-```yaml
-extends:
-  - "example-spectral-ruleset@0.2.0"
+```json
+{
+  "dependencies": {
+    "example-spectral-ruleset": "0.2.0"
+  }
+}
 ```
 
```

Comment:
```suggestion
or through the use of CDNs for NPM repository, such as [unpkg.com](https://unpkg.com/):

```yaml
extends:
  - "https://unpkg.com/example-spectral-ruleset@0.2.0"
``` 
```

I'm not sure if GH will get the formatting right here, so perhaps just copy it over manually.